From a0ba13e28c271a80144a96d6c5e33c538e3b48cf Mon Sep 17 00:00:00 2001
From: luca abeni <luca.abeni@santannapisa.it>
Date: Tue, 16 Jan 2018 11:42:28 +0100
Subject: [PATCH 8/9] Add HCBS extraversion

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 71fd5c2ce067..4550f76affc8 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 5
 PATCHLEVEL = 1
 SUBLEVEL = 0
-EXTRAVERSION = -rc5
+EXTRAVERSION = -rc5-HCBS
 NAME = Shy Crocodile
 
 # *DOCUMENTATION*
-- 
2.19.1

