From 4c23916fee3015444dcab2c68a981d59a0ad50be Mon Sep 17 00:00:00 2001
From: luca abeni <luca.abeni@santannapisa.it>
Date: Tue, 16 Jan 2018 11:42:28 +0100
Subject: [PATCH 8/9] Add HCBS extraversion

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index bad87c4c8117..b44de6cd6404 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 5
 PATCHLEVEL = 2
 SUBLEVEL = 8
-EXTRAVERSION =
+EXTRAVERSION = -HCBS
 NAME = Bobtail Squid
 
 # *DOCUMENTATION*
-- 
2.20.1

