From 9593554b0b4cce4abc6680ef554e2b426f824402 Mon Sep 17 00:00:00 2001
From: luca abeni <luca.abeni@santannapisa.it>
Date: Fri, 16 Feb 2018 10:34:21 +0100
Subject: [PATCH 10/10] Fix compilation with !CONFIG_RT_GROUP_SCHED

---
 kernel/sched/deadline.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/kernel/sched/deadline.c b/kernel/sched/deadline.c
index 95c4bf5bc2ab..05f4ae4a96bb 100644
--- a/kernel/sched/deadline.c
+++ b/kernel/sched/deadline.c
@@ -190,6 +190,7 @@ int dl_check_tg(unsigned long total)
 	return 1;
 }
 
+#ifdef CONFIG_RT_GROUP_SCHED
 int dl_init_tg(struct sched_dl_entity *dl_se, u64 rt_runtime, u64 rt_period)
 {
 	struct rq *rq = container_of(dl_rq_of_se(dl_se), struct rq, dl);
@@ -218,6 +219,7 @@ int dl_init_tg(struct sched_dl_entity *dl_se, u64 rt_runtime, u64 rt_period)
 
 	return 1;
 }
+#endif
 
 void dl_change_utilization(struct task_struct *p, u64 new_bw)
 {
-- 
2.14.1

